apps/kevin-malone/README.md

```diff
@@ -12,27 +12,40 @@ A self reporting bot for the movement model
 ## Setup
 
 1. Clone this repository
-1. Create a virtual environment (`python3.9 -m venv ./.venv`)
-1. Activate the virtual environment (`source ./.venv/bin/activate`)
-1. Install dependencies via pip (`pip install -r requirements-dev.txt`)
-1. Setup environment variables as required in ./.env (see env.sample) for the values which need populating
-1. In a separate directory, clone the [Govrn monorepo](https://github.com/Govrn-HQ/govrn-monorepo), and follow the the setup instructions there. 
+2. Create a [virtual environment](https://python-poetry.org/docs/managing-environments)
+   with python `3.8`.
+3. Install dependencies via pip: `poetry run pip install -r requirements-dev.txt`.
+4. Setup environment variables: `cp env.example .env`
```

Comment:
Can we change this section to "Dev setup"? and just explain what is needed in order to build kevin, and run the tests? We can explain the requirements about postgres, protocol-api server, redis, and api keys/env in a separate "run locally" section. We'd just have to move steps 4 + 5

Reply:
Requirements of protocol api is explained in protocol api's readme.
I've expanded the requirements section and moved the last couple of steps to "Run Locally" Section.